packages/main/src/types/BadgeDesignType.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Defines badge design types.
+ *
+ * @readonly
+ * @enum {string}
+ * @public
+ * @author SAP SE
+ * @alias sap.ui.webc.main.types.BadgeDesignType
+ */
+enum BadgeDesignType {
+	/**
```

Comment:
maybe ColorSet1, ColorSet2, ColorSet3 and ValueState or Semantic?
lets discuss in the meeting